Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -1,36 +1,70 @@
 #include "BatteryStateOfChargeService.h"
+#include <QDebug>
 
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
 }
 
 BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
-: initialStateOfChargePercent_(initialStateOfChargePercent)
+    : initialStateOfChargePercent_(initialStateOfChargePercent)
 {
 }
 
+
 BatteryStateOfChargeService::~BatteryStateOfChargeService()
 {
 }
 
 double BatteryStateOfChargeService::totalAmpHoursUsed() const
 {
-    return 0.0;
+    return initialAhUsed_ + chargeAh_;
 }
 
 bool BatteryStateOfChargeService::isCharging() const
 {
-    return false;
+    if (current_ < 0)
+       return true;
+        return false;
```

Comment:
Even though it wouldn't do much, adding this to an `else` statement is better practice.